syntax = "proto3";

package herd.proto;

import "node.proto";

option cc_enable_arenas = true;

message CircuitGraph {
  repeated InputNode input_nodes = 1;
  repeated OperationNode operation_nodes = 3;
  repeated OutputNode output_nodes = 4;
}